### PR TITLE
feature/17: Updated vs-common-document-files-dependency

### LINKED ISSUE #17: Upade vs-common-files-dependency
Update to use virtual-schema-common-document-files 2.0.0

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #17

### CODE REVIEW

File: ci-isolation/pom.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
```
Comment (morazow) [inline ci-isolation/pom.xml:25]: Do you need Java 8 here?

File: ci-isolation/pom.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            </plugin>
+
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>3.0.0</version>
+                <configuration>
+                    <mainClass>com.myorg.CiIsolationApp</mainClass>
```
Comment (morazow) [inline ci-isolation/pom.xml:33]:
```suggestion
                    <mainClass>com.exasol.CiIsolationApp</mainClass>
```

Comment (morazow) [reply ci-isolation/pom.xml:33]: Let's change it in other package names

File: src/main/java/com/exasol/adapter/document/files/S3RandomAccessInputStream.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    long position = 0;
+
+    S3RandomAccessInputStream(final S3Client s3, final S3ObjectDescription s3ObjectToRead) {
+        this.s3 = s3;
+        this.s3ObjectToRead = s3ObjectToRead;
+    }
+
+    @Override
+    public void seek(final long position) {
+        LOGGER.info("seek");
```
Comment (morazow) [inline src/main/java/com/exasol/adapter/document/files/S3RandomAccessInputStream.java:27]:
```suggestion
        LOGGER.info("Seeked to position '" + position + "');
```

File: src/main/java/com/exasol/adapter/document/files/S3RandomAccessInputStream.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    }
+
+    @Override
+    public long getLength() {
+        return this.s3ObjectToRead.getSize();
+    }
+
+    @Override
+    public int read() {
+        LOGGER.info("singleRead");
```
Comment (morazow) [inline src/main/java/com/exasol/adapter/document/files/S3RandomAccessInputStream.java:43]:
```suggestion
        LOGGER.info("Performing single read at position.");
```

**Review by morazow — CHANGES_REQUESTED** [review]

File: ci-isolation/pom.xml (hunk 1 above)
Comment (jakobbraun) [reply ci-isolation/pom.xml:25]: no. Was just the default

**Review by morazow — APPROVED** [review]

**Pull request merged**